network/server/tchannelthrift/node/service.go

```diff
@@ -74,19 +75,27 @@ type service struct {
 	nowFn   clock.NowFn
 	metrics serviceMetrics
 	health  *rpc.NodeHealthResult_
+	idpool  *ts.Pool
```

Comment:
Might make sense to make `idpool` an interface to be consistent with other pools in the codebase